ts/boot/configurePushNotification.ts

```diff
@@ -4,10 +4,19 @@
 
 import { Alert, PushNotificationIOS } from "react-native";
 import PushNotification from "react-native-push-notification";
+
+import { Store } from "../actions/types";
 import { debugRemotePushNotification, gcmSenderId } from "../config";
+import { updateNotificationsInstallationToken } from "../store/actions/notifications";
 
-function configurePushNotifications() {
+function configurePushNotifications(store: Store) {
   PushNotification.configure({
+    // Called when token is generated
```

Comment:
is there an `onError` handler?

Reply:
Sadly it is not implemented :( Look here https://github.com/zo0r/react-native-push-notification/blob/master/index.js